Conductor

#### Expected Behavior:
A new project is created as per 
#### Actual Behavior:
Command runs with this output:
```
Updating pros-mainline... Done
Applying kernel@3.5.2 [####################################] 100%
Finished applying kernel@3.5.2 to Test
Applying okapilib@4.2.0 [####################################] 100%
Finished applying okapilib@4.2.0 to Test
New PROS Project was created:
PROS Project for v5 at: /home/johnnymcmike/Documents/projects/pros/Test (Test)
Name      Version    Origin
--------  ---------  -------------
kernel    3.5.2      pros-mainline
okapilib  4.2.0      pros-mainline
Building project...
Creating cold package with libc,libm,libpros,okapilib [OK]
Stripping cold package  [DONE]
Section sizes:
   text	   data	    bss	  total	    hex	filename
1.29MB  4.35KB  47.66MB  48.95MB 30f12b2 bin/cold.package.elf
Compiled src/main.cpp [OK]
Adding timestamp [OK]
Linking hot project with ./bin/cold.package.elf and libc,libm,libpros,okapilib [ERRORS]
/home/johnnymcmike/gcc-arm-none-eabi-10.3-2021.07/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: Error: unable to disambiguate: -nostartfiles (did you mean --nostartfiles ?)
collect2: error: ld returned 1 exit status
make: *** [common.mk:239: bin/hot.package.elf] Error 1
```
The `Test` directory is still created and is still populated. Changing the argument from `-nostartfiles` to `--nostartfiles` in the `home/johnnymcmike/.config/pros/templates/kernel@3.5.2/` causes it to not recognize `--nostartfiles` as an argument, and a similar error is produced.

#### System information:
Ubuntu 21.04, arm-none-eabi-gcc version is 10.3.1

PROS Version: 3.2.2

--- a/.gitignore
+++ b/.gitignore
@@ -12,3 +12,4 @@ temp.log
 temp.errors
 .d/
 .clangd/
+.cache/

--- a/common.mk
+++ b/common.mk
@@ -236,7 +236,7 @@ $(HOT_BIN): $(HOT_ELF) $(COLD_BIN)
 
 $(HOT_ELF): $(COLD_ELF) $(ELF_DEPS)
 	$(call _pros_ld_timestamp)
-	$(call test_output_2,Linking hot project with $(COLD_ELF) and $(ARCHIVE_TEXT_LIST) ,$(LD) $(LDFLAGS) $(call wlprefix,-nostartfiles -R $<) $(filter-out $<,$^) $(LDTIMEOBJ) $(LIBRARIES) $(call wlprefix,-T$(FWDIR)/v5-hot.ld $(LNK_FLAGS) -o $@),$(OK_STRING))
+	$(call test_output_2,Linking hot project with $(COLD_ELF) and $(ARCHIVE_TEXT_LIST) ,$(LD) -nostartfiles $(LDFLAGS) $(call wlprefix,-R $<) $(filter-out $<,$^) $(LDTIMEOBJ) $(LIBRARIES) $(call wlprefix,-T$(FWDIR)/v5-hot.ld $(LNK_FLAGS) -o $@),$(OK_STRING))
 	@printf "%s\n" "Section sizes:"
 	-$(VV)$(SIZETOOL) $(SIZEFLAGS) $@ $(SIZES_SED) $(SIZES_NUMFMT)
 

